From 989904a15fffc077e7701f62251b3702dd77eb64 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 25 May 2015 10:04:08 +0800
Subject: [PATCH] MLK-10742 ARM: dts: imx6qp-sabreauto: Enable prefetch for
 the four mxcfbs

This patch enables the prefetch feature for the four mxcfbs.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index e26b9e3..85260e6 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -9,6 +9,22 @@
 #include "imx6q-sabreauto.dts"
 #include "imx6qp.dtsi"
 
+&mxcfb1 {
+	prefetch;
+};
+
+&mxcfb2 {
+	prefetch;
+};
+
+&mxcfb3 {
+	prefetch;
+};
+
+&mxcfb4 {
+	prefetch;
+};
+
 &pcie {
 	reset-gpio = <&max7310_c 5 GPIO_ACTIVE_LOW>;
 	status = "okay";
-- 
1.8.0

